ds-ux-guidelines/package.json

```diff
@@ -1,63 +1,68 @@
 {
-  "name": "cbp-ds-ux-guidelines",
-  "version": "1.0.0",
-  "description": "This is the CBP UX Guidelines",
-  "main": "webpack.config.js",
-  "author": "Creative Services",
-  "license": "ISC",
-  "contributors": [
-    {
-      "name": "David Y. Hodge",
-      "email": "david.y.hodge@cbp.dhs.gov"
-    },
-    {
-      "name": "John D. Higgins",
-      "email": "john.d.higgins@associates.cbp.dhs.gov"
-    },
-    {
-      "name": "Ray Robertson",
-      "email": "ray.robertson@associates.cbp.dhs.gov"
-    }
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/US-CBP/cbp-theme/tree/design-system/ds-ux-guidelines"
-  },
-  "engines": {
-    "node": ">= 8"
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
```

Comment:
Let's update this reflect the ds ux guidelines parameters( author can be Creative Services, ds-ux-guidelines can be the name, it is not private, and the description should match the project.